src/corner_kick/public/main.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * This file defines our Electron main process. This process
+ * is responsible for starting our renderer process.
+ *
+ * To learn more about the Electron process system, visit this link:
+ * https://electronjs.org/docs/tutorial/application-architecture
+ */
+
+const electron = require('electron');
+const app = electron.app;
+const BrowserWindow = electron.BrowserWindow;
+
+const path = require('path');
+const url = require('url');
+const isDev = require('electron-is-dev');
+
+const storage = require('electron-json-storage');
+
+let mainWindow;
+
+function main() {
+  // Needed to enable WebGL for all systems.
+  app.commandLine.appendSwitch('--ignore-gpu-blacklist');
+
+  // Default behaviour: kill process when last window is closed.
+  app.on('window-all-closed', () => {
+    app.quit();
+  });
+
+  // Get last saved window bound from persistent data.
+  storage.get('windowBounds', (error, data) => {
+    if (error) {
+      throw error;
+    } else {
+      createWindow(data);
+    }
+  });
+}
+
+/**
+ * Creates a new rendering process and opens a window with the specified bounds.
+ * @param {*} windowBounds bounds our window should have
+ */
+function createWindow(windowBounds) {
+  mainWindow = new BrowserWindow({
+    height: windowBounds.height || 680,
+    width: windowBounds.width || 900,
+    x: windowBounds.x || undefined,
+    y: windowBounds.y || undefined,
+  });
+
+  // We differentiate whether we are running in production or development mode.
+  // Production mode will be using the prebundled file.
+  // Development mode will be connecting to a localhost web server. This allows us
+  // to have hot-reloading as we modify the source code.
+  mainWindow.loadURL(
+    isDev
+      ? 'http://localhost:8080'
+      : `file://${path.join(__dirname, '../build/index.html')}`,
```

Comment:
> I think this would be a bit more readable if we removed the ternary statement and replaced it with an if expression above. (ternary statements can be hard to read, particuarly when they're split across multiple lines)

Ternary statement was replaced with an if statement
```
    // We differentiate whether we are running in production or development mode.
    // Production mode will be using the prebundled file.
    // Development mode will be connecting to a localhost web server. This allows us
    // to have hot-reloading as we modify the source code.
    if (process.env.ELECTRON_ENV === 'production') {
        mainWindow.loadURL(`file://${path.join(app.getAppPath(), RENDERER_START_FILE)}`);
    } else {
        mainWindow.loadURL(DEV_SERVER_URL);
    }
```